lib/job/arBaseJob.class.php

```diff
@@ -44,12 +44,13 @@ class arBaseJob extends Net_Gearman_Job_Common
    * arObjectMoveJob and the import jobs. The Gearman job server doesn't include a built in
    * system to postpone/schedule jobs so, if multiple jobs from the $avoidParallelExecutionJobs
    * variable bellow are executed at the same time, the late ones will wait, retrying after
-   * the amount of seconds indicated in $waitForRetryTime, until the previous ones are finished.
-   * Due to the limitations of the Gearman job server, the waiting jobs will block the workers
-   * executing them until they are finished.
+   * the amount of seconds indicated in $waitForRetryTime, until the previous ones are finished
+   * or the maximun amount of tries ($maxTries) is reached. Due to the limitations of the Gearman
+   * job server, the waiting jobs will block the workers executing them until they are ended.
    */
   protected $avoidParallelExecutionJobs = array('arObjectMoveJob', 'arFileImportJob');
   protected $waitForRetryTime = 10;
+  Protected $maxRetries = 10;
```

Comment:
It'd be nice if these vars were individually configurable per job type - but that's likely out of scope now with only a few hours left. Perhaps can be addressed if/when the critical section guarding is changed to use db locks at some point.  We may discover a need to tune the timeouts in production if run time varies from the testing system.